### ISSUE TITLE
Generate URL with code parameter programatically

### ISSUE THREAD

**Issue by thefrontendwizard** [seek-oss/playroom#65]
I'd like to generate the url with the code parameter programatically so that I can embed the code examples on the docs directly to the playroom with the code already there

**Comment by hipstersmoothie** [comment]
It seems like it just uses `base64-url`. You can generate code links like so:

```js
import base64url from 'base64-url';

const makeUrl = (code) => {
  const encoded = base64url.encode(code);
  return `your-playroom.com/index.html#?code=${encoded}`;
}

export default makeUrl;
```

**Comment by michaeltaranto** [comment]
This feature is available from [`v0.10.0`](https://github.com/seek-oss/playroom/releases/tag/v0.10.0).

```
import { createUrl } from playroom/utils

createUrl({
  code: <code string to include in url>,
  baseUrl?: <if provided returns absolute url, otherwise relative url>,
});
```
